### PR TITLE
Standard derives for error types

### PR DESCRIPTION
**Pull request by dr-orlovsky**
Based on https://github.com/rust-bitcoin/rust-bech32/pull/49, will be ready for merge once new version of `rust-bech32` will be published

### CODE REVIEW

**Comment by clarkmoody** [comment]
I think this is ready for review / rebase and another run on Travis.

**Comment by dr-orlovsky** [comment]
No need of rebase, it's already on top of master and uses newly released version. Closed & opened to re-run Travis

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bitcoin-bech32"
-version = "0.11.0"
+version = "0.11.1"
```
Comment (sgeisler) [inline Cargo.toml:3]: needs a version bump after the bech32m release

File: Cargo.toml [hunk 2]
```diff
@@ -14,4 +14,4 @@ license = "MIT"
 strict = []
 
 [dependencies]
-bech32 = "0.7.1"
+bech32 = "0.7.3"
```
Comment (sgeisler) [inline Cargo.toml:17]: Needs a bump to at least 0.8.0, maybe we also need a new release to add some derives there? Or maybe that was already included in 0.8.0?

**Review by sgeisler — COMMENTED** [review]
Concept ACK, but needs a rebase. Please remove the draft marker when you think it's ready.

**Comment by dr-orlovsky** [comment]
@sgeisler done

File: Cargo.toml (hunk 2 above)
Comment (clarkmoody) [reply Cargo.toml]: New derives went in before 0.8.0

Comment (dr-orlovsky) [reply Cargo.toml]: Still, having 0.8 will be better, or not?

Comment (clarkmoody) [reply Cargo.toml]: That's right. We depend on 0.8.0 for the Bech32m support from #31 .

**Review by sgeisler — APPROVED** [review]
utACK 5e9fdb755b0df1615e52ac8168b02de7aa6c3c7c

**Comment by dr-orlovsky** [comment]
can we have a release also pls?

**Comment by clarkmoody** [comment]
> can we have a release also pls?

0.12.1 has been released on crates.io

**Pull request merged**